lib/components/app/responsive-webapp.js

```diff
@@ -241,7 +248,27 @@ class RouterWrapper extends Component {
         </div>
       </ConnectedRouter>
     )
+
+    return (
+      auth0Config
+        ? (
+          <Auth0Provider
+            audience={AUTH0_AUDIENCE}
+            scope={AUTH0_SCOPE}
+            domain={auth0Config.domain}
+            clientId={auth0Config.clientId}
+            redirectUri={URL_ROOT}
+            {...auth0Callbacks}
```

Comment:
Please avoid spreading these props. There's only three of them and our goal should be readable, maintainable code.

Reply:
This and turning the auth0 callbacks into actions are addressed in https://github.com/opentripplanner/otp-react-redux/pull/175/commits/0cf1737698da95a83d65cc9a40a696afa9f5db48.